svelte-typescript-app/cypress/e2e/todolist.cy.js

```diff
@@ -12,68 +13,68 @@ describe("Doaa", function () {
   });
 
   it("create item-normal", () => {
-    cy.get("#inputID").type("88");
-    cy.get("#inputTask").type("philo's test");
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("88");
+    homepage.getContentTextBox.type("philo's test");
+    homepage.getCreateBtn.click();
   });
   it("create item-no content", () => {
-    cy.get("#inputID").type("89");
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("89");
+    homepage.getCreateBtn.click();
   });
   it("create item-no ID", () => {
-    cy.get("#inputTask").type("philo's test II");
-    cy.get('[type="create"]').click();
+    homepage.getContentTextBox.type("philo's test II");
+    homepage.getCreateBtn.click();
   });
   it("create item-arabic content", () => {
-    cy.get("#inputID").type("90");
-    cy.get("#inputTask").type("فيلوز تست");
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("90");
+    homepage.getContentTextBox.type("فيلوز تست");
+    homepage.getCreateBtn.click();
   });
   it("create item-pre-existing ID", () => {
-    cy.get("#inputID").type("88");
-    cy.get("#inputTask").type("new Philo's test");
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("88");
+    homepage.getContentTextBox.type("new Philo's test");
+    homepage.getCreateBtn.click();
   });
 
   it("create item string in ID box", () => {
-    cy.get("#inputID").type("sdfsdfsd");
-    cy.get("#inputTask").type("new Philo's test");
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("sdfsdfsd");
+    homepage.getContentTextBox.type("new Philo's test");
+    homepage.getCreateBtn.click();
   });
   //this next test is the way it is as sometimes requests fail when text is entered in a JSON object format,
   //this is trying to make it fail, I am not trying to enter an actual JSON object
   //its like when entering an SQL query into a password texrbox to make it fail
   it("create item-JSON as text", () => {
-    cy.get("#inputID").type("93");
-    cy.get("#inputTask").type('{ID:15, task:"JSON example"}');
-    cy.get('[type="create"]').click();
+    homepage.getIDTextBox.type("93");
+    homepage.getContentTextBox.type('{ID:15, task:"JSON example"}');
+    homepage.getCreateBtn.click();
   });
 
   it("delete using X", () => {
-    cy.get(":nth-child(4) > .btn > i").click();
+    homepage.getDelete4thBtn.click();
   });
   it("delete using delete button", () => {
-    cy.get("#inputID").type("9");
-    cy.get('[type="delete"]').click();
+    homepage.getIDTextBox.type("9");
+    homepage.getDeleteBtn.click();
```

Comment:
Point 3 goes for all tests as their is no assertion to pass the test.
The only code you wrote here was to delete an item, but how did you know that it had been deleted?